### DISCUSSION TITLE
GeminiPro Vision模型会话中提供附带网址的问题时报错

### DISCUSSION THREAD

**Discussion by NuerSir** [lobehub/lobehub#1304] [General Question] [answered]
### 💻 Operating System

macOS

### 📦 Environment

Vercel / Zeabur / Sealos

### 🌐 Browser

Chrome

### 🐛 Bug Description

GeminiPro Vision模型会话时出现如下2个错误：

1. 提供附带网址的问题时报错：
`{
  "error": {
    "message": "[GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com/v1/models/gemini-pro:streamGenerateContent?alt=sse: [400 Bad Request] Please ensure that multiturn requests ends with a user role or a function response."
  },
  "provider": "google"
}`
<img width="800" alt="image" src="https://github.com/lobehub/lobe-chat/assets/6175358/2b32e774-214d-46be-9554-29fcdcf65a61">

2. 传图视觉识别报错如下：

`{
  "error": {
    "message": "[GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com/v1/models/gemini-pro-vision:streamGenerateContent?alt=sse: [400 Bad Request] Multiturn chat is not enabled for models/gemini-pro-vision"
  },
  "provider": "google"
}`

**Comment by arvinxx** [comment, marked as answer]
自己翻译下消息应该就能看明白了吧：

> Please ensure that multiturn requests ends with a user role or a function response.

多轮会话需要以 user / function 类型的消息结尾。


>  Multiturn chat is not enabled for models/gemini-pro-vision

gemini-pro-vision 目前不支持多轮会话
